### DISCUSSION TITLE
How to set max-width for zim-pageview?

### DISCUSSION THREAD

**Discussion by phdpe** [zim-desktop-wiki/zim-desktop-wiki#2035] [Q&A]
I would like to set a max-width for the main text view.  However, I could not get my CSS to work.  

I tried to use GTK inspector and found that `#zim-pageview` is wrapped in `GtkScrolledWindow` but it does not have a name.  Setting `GtkScrolledWindow` width in css does not change anything.  

Can anyone point me to the right direction?  Essentially I want to do something like this:

```css
#zim-pageview {
    max-width: 780px;
}
```

**Comment by introt** [comment]
Not sure if it can be done via CSS; for fullscreen, there's the [Distraction Free Editing plugin](https://zim-wiki.org/manual/Plugins/Distraction_Free_Editing.html)

**Reply by phdpe** [reply]
> Not sure if it can be done via CSS; for fullscreen, there's the [Distraction Free Editing plugin](https://zim-wiki.org/manual/Plugins/Distraction_Free_Editing.html)

Thank you for your reply.  I am aware of this plugin, and actually looked into its source code to see if there is anything I can use.  Unfortunately, I cant make the width fixed for non-fullscreen.  

If there is any other way, such as a plugin, please let me know.

**Reply by kyrathasoft** [reply]
I'm seeking an answer to this same question.

**Reply by tristion** [reply]
There's a now a 3rd party plugin called [Textview Preferences](https://omacronides.com/projets/zim-plugin-textview-prefs/)

This does what you're looking for.
